guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>Next, we will concatenate the address into one column and rename the <code>Name</code> column to <code>loc_id</code> to more closely match the example from the first GeoEnrichment notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>